app/packs/src/apps/mydb/elements/details/wellplates/analysesTab/WellplateDetailsContainers.js

```diff
@@ -174,8 +133,51 @@ export default class WellplateDetailsContainers extends Component {
       )
     }
 
-  }
+    var analyses_container = wellplate.container.children.filter(element => ~element.container_type.indexOf('analyses'));
+
+    if (analyses_container.length != 1 || analyses_container[0].children.length == 0) {
+      return (
+        <div>
+          <p className='noAnalyses-warning'>
+            There are currently no Analyses.
+            {this.addButton()}
+          </p>
+        </div>
+      )
+    }
 
+    return (
+      <div>
+        <div className="my-3">
+          {this.addButton()}
+        </div>
+        <Accordion defaultActiveKey={0}>
+          {
+            analyses_container[0].children.map((container, key) => {
+              return (
```

Comment:
NIT: You might as well get rid of this `return` by using the arrow function expression:
 `…map((...) => (<Comp />))`

Reply:
I know but I decided not to